heron/packing/tests/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPackingTest.java

```diff
@@ -81,6 +81,35 @@ private static PackingPlan getFirstFitDecreasingPackingPlanRepack(
     return packing.repack(currentPackingPlan, componentChanges);
   }
 
+  PackingPlan doScalingTest(TopologyAPI.Topology t,
```

Comment:
You could make the method static and pass `totalInstances` in as `totalExpectedInstances`